### ISSUE TITLE
Abandonware?

### ISSUE THREAD

**Issue by dhpollack** [xuedi/xuedi#7]
No updates in a while.  Should this repo be considered abandoned and archived?

**Comment by xuedi** [comment]
Yeah, the quality is quite trashy, it not functioning anymore the intended ware ... should be properly deleted ...

**Comment by dhpollack** [comment]
Pro-move, abandon the issues on the abandoned repo and never close them.
